app/services/bulk_unsubscribe_service.rb

```diff
@@ -58,6 +58,7 @@ def self.call(
                                 .subscriber_list
                                 .links
                                 .values
+                                .flat_map(&:values)
                                 .flatten
```

Comment:
The links hash is something like:
```links: { topics: { any: [t1, t2] }, taxon_tree: {any: [c1, c2] } }```

So links.values = ```[{ any: [t1, t2] },  {any: [c1, c2] }]```
links.values.flat_map(&:values) = ```[[t1, t2], [c1, c2]]```
so the flatten is still necessary to make a single array.

It might need a bit more clarification, but this is code we wrote in the last quarter to email people subscribed to policy areas and would have to be modified if ever reused anyway so I didn't want to spend too much time on it.